library/nis.po

```diff
@@ -41,37 +46,49 @@ msgid ""
 "Return value is an arbitrary array of bytes (may contain ``NULL`` and other "
 "joys)."
 msgstr ""
+"Renvoie le résultat de *clé* dans carte *mapname*, ou lève une erreur (:exc:"
+"`nis.error`) s'il n'y en a pas. Toutes les deux doivent être des chaînes, "
+"*key* est une 8-bit propre. Renvoie valeur est un tableau arbitraire "
+"d'octets (pourrait contenir ``ǸULL`` et autres joyeusetés). "
 
 #: ../Doc/library/nis.rst:29 ../Doc/library/nis.rst:41
 msgid "Note that *mapname* is first checked if it is an alias to another name."
 msgstr ""
+"Notez que *mapname* est vérifié la première fois si c'est un alias d'un "
+"autre nom."
 
 #: ../Doc/library/nis.rst:31 ../Doc/library/nis.rst:43
 #: ../Doc/library/nis.rst:51
 msgid ""
 "The *domain* argument allows overriding the NIS domain used for the lookup. "
 "If unspecified, lookup is in the default NIS domain."
 msgstr ""
+"L'argument *domain* permet de passer outre le domaine NIS utilisé pour les "
+"recherche (lookup). Quand non spécifié, recherche est dans le domaine NIS "
+"défaut "
 
 #: ../Doc/library/nis.rst:37
 msgid ""
 "Return a dictionary mapping *key* to *value* such that ``match(key, "
 "mapname)==value``. Note that both keys and values of the dictionary are "
 "arbitrary arrays of bytes."
 msgstr ""
+"Renvoie un dictionnaire qui associe *key* à *value* tel que ``match(key, "
+"mapname)==value``. Notez que les deux clés et valeurs du dictionnaire sont "
```

Comment:
Je crois que `both keys and values` se traduit plutôt par `à la fois les clés et les valeurs`.

Du coup quelque chose comme ça peut-être ?

```
Notez que et les clés, et les valeurs du dictionnaires sont des...
```

Reply:
:+1: